packages/host/app/components/mode-switcher.gts

```diff
@@ -0,0 +1,138 @@
+import Component from '@glimmer/component';
+import { BoxelDropdown, Button, Menu } from '@cardstack/boxel-ui';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+import { menuItemFunc, MenuItem } from '@cardstack/boxel-ui/helpers/menu-item';
+import { fn } from '@ember/helper';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import map from 'ember-composable-helpers/helpers/map';
+import { service } from '@ember/service';
+import RouterService from '@ember/routing/router-service';
+import type OperatorModeStateService from '../services/operator-mode-state-service';
+import type CardService from '../services/card-service';
```

Comment:
Is this used?

Reply:
updated [9040e1b](https://github.com/cardstack/boxel/pull/554/commits/9040e1bf8c282c3481e69e7e6a56d8a1be235391)